quick-integration.hcl

```diff
@@ -0,0 +1,58 @@
+param "converge-bin-dir" {}
+
+param "image-name" {
+  default = "ubuntu"
+}
+
+param "image-tag" {
+  default = "latest"
+}
+
+docker.container "converge" {
+  name  = "converge-test"
+  image = "{{param `image-name`}}:{{param `image-tag`}}"
+  status = "running"
+
+  expose = ["4774"]
+  publish_all_ports = "true"
+
+  volumes = [
+    "/converge",
+    "{{env `PWD`}}/{{param `converge-bin-dir`}}:/converge/bin",
+    "{{env `PWD`}}/samples:/converge/samples",
+  ]
+
+  command = ["/converge/bin/converge", "server", "--no-token"]
+}
+
+task.query "wait" {
```

Comment:
once #334 is merged, this could be replaced to wait for the rpc port to be available.